.github/CODEOWNERS

```diff
@@ -35,3 +35,17 @@
 /infra            @ethereum-optimism/infra-reviewers
 /specs            @ethereum-optimism/contract-reviewers @ethereum-optimism/go-reviewers
 /endpoint-monitor @ethereum-optimism/infra-reviewers
+
+# Don't add owners if only package.json is updated
+/packages/*/package.json 
+/*/package.json 
+
+# JavaScript Releases
+
+# Have release-reviewers review releases after the group exists
+# /packages/*/CHANGELOG.md @ethereum-optimism/release-reviewers
+# /*/CHANGELOG.md          @ethereum-optimism/release-reviewers
```

Comment:
I don't have permission to actually add this group but we should create a release-reviewers group